keyboards/keychron/c3_pro/c3_pro.c

```diff
@@ -16,6 +16,28 @@
 
 #include "c3_pro.h"
 
+void keyboard_post_init_kb(void) {
+    gpio_set_pin_output_push_pull(LED_MAC_OS_PIN);
+    gpio_set_pin_output_push_pull(LED_WIN_OS_PIN);
+    gpio_write_pin(LED_MAC_OS_PIN, !LED_OS_PIN_ON_STATE);
+    gpio_write_pin(LED_WIN_OS_PIN, !LED_OS_PIN_ON_STATE);
+    
+    keyboard_post_init_user();
+}
+
+void housekeeping_task_kb(void) {
+    if (default_layer_state == (1U << 0)) {
+        gpio_write_pin(LED_MAC_OS_PIN, LED_OS_PIN_ON_STATE);
+        gpio_write_pin(LED_WIN_OS_PIN, !LED_OS_PIN_ON_STATE);
+    }
+    if (default_layer_state == (1U << 2)) {
+        gpio_write_pin(LED_MAC_OS_PIN, !LED_OS_PIN_ON_STATE);
+        gpio_write_pin(LED_WIN_OS_PIN, LED_OS_PIN_ON_STATE);
+    }
```

Comment:
Align to same layer notation used in `KC_OSSW`

```suggestion
    if (get_highest_layer(default_layer_state) == 0) {
        gpio_write_pin(LED_MAC_OS_PIN, LED_OS_PIN_ON_STATE);
        gpio_write_pin(LED_WIN_OS_PIN, !LED_OS_PIN_ON_STATE);
    }
    if (get_highest_layer(default_layer_state) == 2) {
        gpio_write_pin(LED_MAC_OS_PIN, !LED_OS_PIN_ON_STATE);
        gpio_write_pin(LED_WIN_OS_PIN, LED_OS_PIN_ON_STATE);
    }
```